src/SFML/Graphics/Texture.cpp

```diff
@@ -120,6 +121,49 @@ Texture::~Texture()
     }
 }
 
+////////////////////////////////////////////////////////////
+Texture::Texture(Texture&& right) noexcept :
+m_size{std::exchange(right.m_size, {0U, 0U})},
```

Comment:
```suggestion
m_size(std::exchange(right.m_size, {0U, 0U})),
```
Please use parentheses for initialization as per SFML convention